content/docs/user-guide/troubleshooting.md

```diff
@@ -91,3 +91,17 @@ using:
 ```dvc
 $ dvc checkout --relink
 ```
+
+## HTTP Git authentication is not supported {#git-auth}
+
+[Experiment sharing](/doc/user-guide/experiment-management/sharing-experiments)
+commands accept a `git_remote` argument. In order to access the Git remote,
```

Comment:
This is not just about `git_remote` right? Even if that's not provided, the default remote should also use SSH auth (if auth is needed).

Reply:
This error will only happen in commands using `git_remote`. Other errors regarding DVC remote auth would depend on the remote type used and will be handled with a different exception and error message like [`GDriveAuthError`](https://github.com/iterative/dvc/blob/master/dvc/fs/gdrive.py)

Reply:
Hmm that sounds a bit strange but OK. In any case the problem itself is broader right? It also applies to configured remotes so why not explain it in it's broader context here? Even if this particular troubleshooting section is only reached (now) via `exp push/list/push <git_remote>`

No strong opinion but I think it would be make sense to make it broader.

Reply:
I think that the thing is that, for example, the DVC HTTP remote does ask for users/password (hence my comment that errors for each remote should be handled with separated exceptions). The `git_remote` parts of `dvc exp` do not share code with `dvc remote`.

Reply:
Hmmm. OK it sounds like this is closely related with remote config per https://github.com/iterative/dvc.org/pull/2731#pullrequestreview-751064480 so I still think we could explain the broader context but maybe I'm not getting why that's not a good option. But OK